package.json

```diff
@@ -20,7 +21,8 @@
     "@salesforce/plugin-trust": "^3.7.89",
     "@salesforce/sf-plugins-core": "^12.2.5",
     "@salesforce/ts-types": "^2.0.10",
-    "aws-sdk": "^2.1692.0",
+    "@smithy/node-http-handler": "^4.4.5",
+    "@smithy/types": "^4.8.1",
```

Comment:
Technically, the most recent version of this module is `4.9.0`, but using that version causes type-mismatches with `aws-sdk-client-mock`, so I set it to use the version that the mocking library uses.